src/index.html

```diff
@@ -11,7 +11,96 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surname:</label>
+          <input id="surname" type="text"
+          autocomplete="off">
+        </div>
+        <div>
+          <label for="name">Name:</label>
+          <input id="name" type="text"
+          autocomplete="off" minlength="2">
+        </div>
+        <div>
+          <label for="age">How old are You?</label>
+          <input id="age" type="number"
+          min="1" max="100"
+          value="12">
+        </div>
+        <div id="one">
+          <label for="date">Full date of birth:</label>
+          <input id="date" type="date">
+        </div>
+        <div>
+          <label for="acceptance">I accept the term of the agreement</label>
+          <input id="acceptance" type="checkbox">
+        </div>
+      </fieldset>
+      <fieldset class="fields">
```

Comment:
It's good practice to add empty lines between multiline sibling elements. 
```suggestion
      </fieldset>
      
      <fieldset class="fields">
```

Reply:
solved.